ergotree-interpreter/src/eval/byte_array_to_bigint.rs

```diff
@@ -0,0 +1,140 @@
+use ergotree_ir::mir::byte_array_to_bigint::ByteArrayToBigInt;
+use ergotree_ir::mir::value::Value;
+
+use crate::eval::env::Env;
+use crate::eval::EvalContext;
+use crate::eval::EvalError;
+use crate::eval::EvalError::UnexpectedValue;
+use crate::eval::Evaluable;
+use ergotree_ir::mir::constant::TryExtractInto;
+use ergotree_ir::mir::value::Value::BigInt;
+use lazy_static::lazy_static;
+use num_bigint::ToBigInt;
+use num_traits::pow::Pow;
+
+impl Evaluable for ByteArrayToBigInt {
+    fn eval(&self, env: &Env, ctx: &mut EvalContext) -> Result<Value, EvalError> {
+        let input = self.input.eval(env, ctx)?.try_extract_into::<Vec<u8>>()?;
+        if input.is_empty() {
+            return Err(UnexpectedValue(
+                "ByteArrayToBigInt: byte array is empty".into(),
+            ));
+        }
+        let n = num_bigint::BigInt::from_signed_bytes_be(&input);
+        println!("{}", n);
+        println!("{:x}", n);
```

Comment:
I assume these println!() calls were for debugging purposes and are no longer needed.

Reply:
Yes. They were to test that tests test what they supposed to test